requirements.txt

```diff
@@ -1,5 +1,5 @@
-mkdocs==1.4.1
-mkdocs-material==8.5.7
+mkdocs==1.5.2
+mkdocs-material==9.2.6
```

Comment:
Such changes should rather go into separate PR and undergo more thorough verification if something broke.

Reply:
Alright, moved that change to https://github.com/Dasharo/docs/pull/645 and changed this PR's base to that PR.